src/cattr/_compat.py

```diff
@@ -101,3 +114,6 @@ def is_bare(type):
         return not type.__args__
 
     is_bare_frozenset = is_bare
+
+    def get_type(ignored: type, type_: type) -> type:
```

Comment:
same here: `type` without underscore suffix as arg name, `Type` (from the `typing` module) for the annotations.